bitsail-connectors/connector-rocketmq/src/main/java/com/bytedance/bitsail/connector/rocketmq/source/RocketMQSource.java

```diff
@@ -91,6 +92,15 @@ public String getReaderName() {
   public ParallelismAdvice getParallelismAdvice(BitSailConfiguration commonConfiguration,
                                                 BitSailConfiguration rocketmqConfiguration,
                                                 ParallelismAdvice upstreamAdvice) throws Exception {
+    if (rocketmqConfiguration.fieldExists(ReaderOptions.BaseReaderOptions.READER_PARALLELISM_NUM)) {
```

Comment:
Need to check if reader_parallelism_num is valid, _i.e._, >0 and < num_of_queues.

Reply:
Solved.